PSReadLine.build.ps1

```diff
@@ -325,7 +325,9 @@ task LayoutModule BuildMainModule, BuildMamlHelp, {
 
     foreach ($file in $extraFiles)
     {
-        Copy-Item $file $targetDir
+        # ensure files have \r\n line endings as the signing tool only uses those endings to avoid mixed endings
```

Comment:
The line endings depend on your git configuration. It is possible to check out with with either style on both Windows and Unix.

Because of this, I don't think it's safe to assume LF endings - you don't want to replace CRLF with CRLFLF.

This code might also inadvertently change a binary file.

One possible fix is to take a dependency on the `unix2dos` command, installed on Ubuntu with `sudo apt install dos2unix`. On Windows, we could also expect it's installed, possibly in wsl, so e.g. running `wsl unix2dos $file`.

Reply:
Perhaps I can have a whitelist of just the text files and also check if the file contains CRLF before doing the conversion.  Prefer not to add another dependency.